core/messages/auth/en.ts

```diff
@@ -7,34 +7,34 @@ export const authMessages = i18n('auth', {
   createAccount: 'Create cross-device account',
   customServer: 'Use my own server',
   email: params(
-    '// Send this email to yourself\n// If you will forget password, you will search by it here\n\nUser ID: {user}\nSecret: {secret}'
+    '// Send this email to yourself\n// If you forget your password, you can search for it later here\n\nUser ID: {user}\nSecret: {secret}'
   ),
-  localDescription1: 'Slow Reader works locally.',
+  localDescription1: 'Slow Reader works right on your device — no account needed.',
   localDescription2:
-    'You can create account later when you will need cross-device sync.',
-  login: 'Login into existing account',
-  newUser: 'New user',
+    'You can create an account later to sync across devices.',
+  login: 'Log in',
+  newUser: 'Get started locally',
   noRecoveryDesc:
-    'The app uses end-to-end encryption. But if you will lose your password, nobody will be able to decrypt it.',
-  noRecoveryTitle: 'We don’t have password recovery',
-  oldUser: 'Existing user',
+    'Slow Reader uses end-to-end encryption. If you lose your password, no one will be able to decrypt your data.',
```

Comment:
```suggestion
    'Slow Reader uses end-to-end encryption. If you lose your password, no one will be able to decrypt your data.',
```

I added non-break space to fix line breaks